medical/models.py

```diff
@@ -278,7 +282,47 @@ def save_history_on_update(sender, instance, **kwargs):
         instance.validity_from = now
 
 
-class ItemMutation(core_models.UUIDModel, ObjectMutation):
+class ServiceService(models.Model):
```

Comment:
Should use businessHistoryModel instead of Model to have auditing and timecoorelation